Userland/Applications/Mail/AccountHolder.cpp

```diff
@@ -74,3 +74,28 @@ void AccountHolder::rebuild_tree()
 {
     m_mailbox_tree_model->invalidate();
 }
+
+void MailboxNode::update_display_name_with_unseen_count()
+{
+    m_display_name_with_unseen_count = ByteString::formatted("{} ({})", m_display_name, m_unseen_count);
+}
+
+void MailboxNode::decrement_unseen_count()
+{
+    if (!m_unseen_count)
+        return;
+    --m_unseen_count;
```

Comment:
How about replacing the next 3 lines with `set_unseen_count(m_unseen_count - 1);`? (Similar for the increment method.)

Reply:
done, and updated my initial comment with a screenshot